example-plugins/sethome.lua

```diff
@@ -0,0 +1,94 @@
+local lukkit = require("lukkit")
```

Comment:
Split up?
```lua
local lukkit = require('lukkit')
local plugin = require('plugin')
```

Reply:
Granularity never hurts IMO, also clear separation of concerns

Reply:
Would probably only split it at the module level (e.g. `http`, `store`...). The global `lukkit` would be the bare essentials to get a plugin running such as lifecycle, commands etc..

Reply:
I think instead of calling it `lukkit` we just call it `plugin`

Reply:
I think instead of calling it `lukkit` we just call it `plugin`

Reply:
Would rather call the core `lukkit` or `lukkit.core` and have submodules as `lukkit.<submodule>`. Keeps it distinguished from user-defined files.

Reply:
Alright, sounds good.